pkg/backup/util.go

```diff
@@ -99,3 +100,8 @@ func toMB(s int64) float64 {
 	}
 	return n
 }
+
+// ToPrefix concatenates s3Prefix, "v1", namespace, clusterName to a single prefix path.
+func ToPrefix(s3Prefix, namespace, clusterName string) string {
+	return path.Join(s3Prefix, "v1", namespace, clusterName)
```

Comment:
Maybe add `S3V1="v1"` to "backupapi/" pkg ?

This func should also be called ToS3Prefix ?